README.md

```diff
@@ -230,7 +215,9 @@ See `apps/test-logger` for an example of using the SWO-supported logger utility.
 
 ### Ubuntu
 
-**TODO**
+1. In one terminal, launch the openocd server and specify runtime parameters with 
+`openocd -f /usr/share/openocd/scripts/interface/stlink-v2.cfg -f /usr/share/openocd/scripts/target/stm32f4x.cfg -c "tpiu config internal - uart off 180000000" -c "itm ports on"`
```

Comment:
this cmd will only work for programming an stm32f4 with an st link v2. i thought id add a warning but figured it might just confuse ppl bc we exclusively use these twwo

Reply:
Also its pretty obvious from the arguments that this cmd is only meant for f4 and stlinkv2

Reply:
so doesnt work for flashing from nucleo stlink?

Reply:
so doesnt work for flashing from nucleo stlink?

Reply:
oops my comment was misleading. this cmd is for printing stuff and it'll only work for f4 and st link v2. so it doesnt make sense to run this cmd on nucleo bc it doesnt print via swo.

I'll check if we can flash the nucleo with st-flash cmd

Reply:
Doesnt the nucleo have a swo interface we can print to?

Reply:
ah yes true. ig we should standardize things and make all our prints go through SWO and not UART. i'll look into this tomorrow prob. As the older revs become less relevant i think we should remove code that prints over UART (assuming the stuff im checking tomorrow goes well)

Reply:
probs not worth it rn bcuz of sar priorities